### PR TITLE
Only keep two random static nodes from fleets.status.im

### PR DESCRIPTION
**Pull request by pedropombeiro**

### CODE REVIEW

**Review by dshulyak — APPROVED** [review]

**Comment by dshulyak** [comment]
thanks. i forgot to generate static in my pr

File: _assets/ci/update-fleet-config.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ for fleet in ${fleets[@]}; do
     fleetJSON=$(echo $json | jq ".fleets.\"$fleet\"")
     boot=$(echo $fleetJSON | jq ".boot | map(.)" -r)
     mail=$(echo $fleetJSON | jq ".mail | map(.)" -r)
-    whisper=$(echo $fleetJSON | jq ".whisper | map(.)" -r)
+    whisper=$(echo $fleetJSON | jq ".whisper | map(.) | .[0:2]" -r)
```
Comment (adambabik) [inline _assets/ci/update-fleet-config.sh]: It would be nice to randomize nodes selection.

**Review by adambabik — APPROVED** [review]

**Comment by pedropombeiro** [comment]
OK, adapted the script to pick 2 random nodes from fleet

File: _assets/ci/update-fleet-config.sh [hunk 2]
```diff
@@ -12,7 +12,13 @@ for fleet in ${fleets[@]}; do
     fleetJSON=$(echo $json | jq ".fleets.\"$fleet\"")
     boot=$(echo $fleetJSON | jq ".boot | map(.)" -r)
     mail=$(echo $fleetJSON | jq ".mail | map(.)" -r)
-    whisper=$(echo $fleetJSON | jq ".whisper | map(.)" -r)
+
+    # Get random nodes from whisper node list
+    maxStaticNodeCount=2
+    staticNodeCount=$(echo $fleetJSON | jq ".whisper | length")
+    index=$(($RANDOM % ($staticNodeCount - ($maxStaticNodeCount - 1))))
```
Comment (adambabik) [inline _assets/ci/update-fleet-config.sh:19]: Where does `$RANDOM` come from?

Comment (pedropombeiro) [reply _assets/ci/update-fleet-config.sh:19]: http://tldp.org/LDP/abs/html/randomvar.html

**Pull request merged**